libraries/AP_HAL_SITL/Util.cpp

```diff
@@ -138,3 +139,10 @@ enum AP_HAL::Util::safety_state HALSITL::Util::safety_switch_state(void)
     }
     return sitl->safety_switch_state();
 }
+
+void HALSITL::Util::set_cmdline_parameters()
+{
+    for (auto param: sitlState->cmdline_param) {
+        AP_Param::set_and_save_by_name(param.name, param.value);
```

Comment:
why not set_default_by_name()?
I don't think we want the command line options to persist to the next run, where the user may not provide the option at all

Reply:
done